packages/manager/modules/pci/src/projects/project/storages/databases/database/current-queries/current-queries.html

```diff
@@ -123,5 +123,15 @@ <h2 data-translate="pci_databases_current_queries_title"></h2>
         >
             <span data-ng-bind="$value ? $value : '-'"></span>
         </oui-datagrid-column>
+        <oui-action-menu data-compact data-placement="end">
+            <oui-action-menu-item
+                data-on-click="$ctrl.trackAndTerminateQuery($row)"
+                data-disabled="$row.state === 'IDLE'"
```

Comment:
Can we use a constant for queries statuses ?

Reply:
Fixed :)